src/renderers/dom/stack/client/wrappers/ReactDOMInput.js

```diff
@@ -122,6 +123,7 @@ var ReactDOMInput = {
       initialChecked: props.checked != null ? props.checked : props.defaultChecked,
       initialValue: props.value != null ? props.value : defaultValue,
       listeners: null,
+      onBlur: _handleBlur.bind(inst),
```

Comment:
This won't get called anymore. Though I'm unclear why. I had thought this function was a filter/transform step for props before they entered the component. When I follow the stack trace for the blur event, this doesn't seem to be the case.